### ISSUE TITLE
Members list not retrieved in OBJECT BROWSER for one file

### ISSUE THREAD

**Issue by Hurrican39** [halcyon-tech/vscode-ibmi#1414]
Type: <b>Bug</b>

Opening libraries with only *SRCPF files beginning with Q
Files are correctly displayed.
When trying to unfold one specific file (QRPGLESRC in our ELSOCO library), it does nothing, no member list is displayed .
All other files (and QRPGLESRC in other libraries), are displaying members list correctly.
I searched but found nothing specific to this file (except it's the most heavy source file, 756 members, 54MB on disk).

Extension version: 1.9.4
VS Code version: Code 1.79.2 (695af097c7bd098fbf017ce3ac85e09bbc5dda06, 2023-06-14T08:58:52.392Z)
OS version: Darwin x64 19.6.0
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|2, 1, 1|
|Memory (System)|16.00GB (0.11GB free)|
|Process Argv||
|Screen Reader|no|
|VM|15%|
</details>

**Comment by Hurrican39** [comment]
Hi,
All members use A-Z letters and 0-9 numbers in their names.
Two are using underscore "_"  character, but other source files with members using underscore are correctly displayed in the list.

Note, I can still access members by selecting "Go to file", but they are not displayed in the object browser list.

**Comment by Hurrican39** [comment]
Finally found it !
In the output, I saw there was a date problem.
I just run the same SQL request, in SQL scripts in ACS and changed  cast (for bit data) to standard fields to be able to read the output.
I had 8 members without creation date ! 
Why ? That's a good question. But I copied those members to brand new ones, deleted the ones with missing creation date, renamed the new members to the original name, and now the list is correctly displayed.

May be the SQL request could be updated to replace missing creation dates by a default value ?

Thanks.
